app/assets/stylesheets/farmbot.css.scss

```diff
@@ -120,7 +148,7 @@ $brown:          #CA8;
     height: 3rem;
     max-width: 98%;
   }
-  padding: 0 0 0 5px;
+  padding: 0 5px 0 5px;
```

Comment:
Properties should be ordered color, padding<br>Shorthand form for property `padding` should be written more concisely as `0 5px` instead of `0 5px 0 5px`